cmake/Modules/FindAubio.cmake

```diff
@@ -1,8 +1,10 @@
 
 if (NOT USE_SELF_BUILT_AUBIO)
     find_package(PkgConfig REQUIRED)
-    # The version we look for is legacy, may need some fine tunning
-    pkg_check_modules(AUBIO IMPORTED_TARGET aubio>=0.4.9)
+    if(PKG_CONFIG_FOUND)
```

Comment:
Shouldn't we take away the word `REQUIRED` from previous line? I thought it will be an error if `REQUIRED` option is set...

But to be honest I don't wrote code that doesn't requires libraries or packages :)

Reply:
This `FindAubio.cmake` module should be adjusted in a whole by using our macro for `pkg_check_modules`. This won't be addressed within this PR though.